datasources/nm.ejs

```diff
@@ -17,7 +17,7 @@
   ],
   "catalog": [
         {
-            "name": "National Data Sets",
+            "name": "National Datasets",
```

Comment:
Nevermind, Alex set me straight, so I've added the necessary properties for backward compatibility and merged it with the renames.